.travis.yml

```diff
@@ -0,0 +1,27 @@
+sudo: required
+
+services:
+  - docker
+
+language: python
+
+python:
+  - "3.6"
```

Comment:
Should this be 3.7?

Reply:
Done, also required adding `dist: xenial` to use python versions >= 3.7